hack/verify-manifest.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+./hack/generate-manifests.sh
+
+if [[ -n $(git status -s assets/) ]]; then
+    echo 'Assets has been modified and/or untracked'
+    exit -1
```

Comment:
* `git diff` would be nice to see the differences
* suggest "ERROR: generated kustomize manifests need to be updated. Run `make update`", so 1) CI result highlights `ERROR:` and 2) people outside of our team know what to do.